### PR TITLE
Move tool into new tool subdir and rename it to fc4-tool

### PR DESCRIPTION
**Pull request by aviflax**
As explained in #58.

This will be followed up by 2 PRs: one that will merge all the commits from the `fc4-framework` repo into this one, and another that will update the name of the tool in the files brought in via that merge.

### CODE REVIEW

File: tool/README.md [hunk 1]
```diff
@@ -43,18 +43,18 @@ As explained in [The Authoring Workflow](https://fundingcircle.github.io/fc4-fra
 [the FC4 Methodology](https://fundingcircle.github.io/fc4-framework/methodology/):
 
 > 1. In your text editor: either create a new diagram source file or open an existing diagram source file
-> 1. In a terminal, in your FC4C working dir, run `./wcb`
+> 1. In a terminal, in your fc4-tool working dir, run `./wcb`
```
Comment (sldblog) [inline tool/README.md]: Would this be `tool/wcb` when it's merged?

Comment (aviflax) [reply tool/README.md]: Yes! Good catch! Thanks!

**Comment by aviflax** [comment]
Thanks Conor!

File: tool/README.md (hunk 1 above)
Comment (aviflax) [reply tool/README.md]: Fixed in e6e437e

**Pull request merged**